rotkehlchen/tests/unit/test_eth2.py

```diff
@@ -882,3 +887,47 @@ def test_validator_daily_stats_with_db_interaction(  # pylint: disable=unused-ar
             )
         last_stat = stats[:len(expected_stats)][-1]
         assert last_stat.pnl_balance.amount == expected_stats[-1].pnl_balance.amount * FVal(0.45)  # noqa: E501
+
+
+@pytest.mark.parametrize('default_mock_price_value', [FVal(1.55)])
+def test_validator_daily_stats_with_event_in_first_day(
+        network_mocking: bool,
+        price_historian: 'PriceHistorian',  # pylint: disable=unused-argument
+        function_scope_messages_aggregator: 'MessagesAggregator',
+) -> None:
+    """
+    Test that if profit returned by beaconchain on first day is > 32 ETH the app
```

Comment:
edit comment: I think we are talking about genesis day. Not just validator's first deposit day